crates/host/src/wasmbus/mod.rs

```diff
@@ -49,9 +49,10 @@ use tracing::{debug, error, info, instrument, trace, warn};
 use ulid::Ulid;
 use uuid::Uuid;
 use wascap::{jwt, prelude::ClaimsBuilder};
+#[allow(deprecated)]
```

Comment:
Please add a comment about what is deprecated, when we'll remove it and ideally add an issue ref as well

Reply:
If that's just for the `start_actor`, then can we remove the import here and refer to it using absolute path to avoid having to do this?